packages/design/.storybook/config.js

```diff
@@ -14,11 +14,21 @@ const reqDC = require.context(
     /\.stories\.tsx$/,
 );
 
+const viewportOptions = {
+    name: 'Responsive',
+    styles: {
+        width: '100%',
+        height: '100%',
```

Comment:
will this scroll up/down?

Reply:
I don't think these options do anything tbh

Reply:
1 sec

Reply:
Removed it